CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ==========
 
+
+0.7.0 (2023-07-12)
```

Comment:
The date might need to be adjusted unless this PR is accepted quickly. Or the package maintainers might want to handle this part of releasing themselves? Just tell me how you want it. I went with the path where this PR does everything needed before `cargo publish` for now, to make it easy :)